### PR TITLE
Feature: Commands API

### LINKED ISSUE #26: Add Support for Commands API
Add support for the [Commands API](https://m2x.att.com/developer/documentation/v2/commands).

### PR DESCRIPTION
**Pull request by fsaravia**
Closes #26

### CODE REVIEW

**Comment by fsaravia** [comment]
@stream7: This is ready for review!

File: lib/devices.js [hunk 1]
```diff
@@ -424,6 +424,50 @@ Devices.prototype.log = function(id, callback) {
...
+Devices.prototype.command = function(id, commandId, callback) {
+    return this.client.get(
+        helpers.url("/devices/%s/commands/%s", id, commandId),
+        callback);
+};
+
+// Process a command
+//
+// https://m2x.att.com/developer/documentation/v2/commands#Device-Marks-a-Command-as-Processed
+Devices.prototype.process_command = function(id, commandId, callback) {
```
Comment (stream7) [inline lib/devices.js]: Shall we use `camelCase`?

File: lib/devices.js [hunk 2]
```diff
@@ -424,6 +424,50 @@ Devices.prototype.log = function(id, callback) {
...
+    return this.client.post(
+        helpers.url("/devices/%s/commands/%s/process", id, commandId),
+        callback);
+};
+
+// Reject a command
+//
+// https://m2x.att.com/developer/documentation/v2/commands#Device-Marks-a-Command-as-Rejected
+Devices.prototype.reject_command = function(id, commandId, callback) {
```
Comment (stream7) [inline lib/devices.js]: Same here. Other than that it looks :+1:

File: lib/devices.js (hunk 1 above)
Comment (fsaravia) [inline lib/devices.js]: My bad, thanks for noticing!

**Pull request merged**